TACC/Core-Styles
# fix: style non-footer form buttons (and add demo to test)

## Overview & Changes

- Style buttons **not** in a form footer.
- Demo buttons **not** in a form footer.
- Demo reset button **in** a form footer.
- Change selector for buttons **in** a form footer.

## Related

- required by https://github.com/tapis-project/authenticator/pull/54

## Testing

On an `s-form` and a `c-form`:

1. Verify buttons **in** a form footer are styled:
	| type | style |
	| - | - |
	| `submit` | primary (purple) |
	| `button` | secondary (gray) |
	| `reset` | tertiary (white) |
2. Verify buttons **not** in a form footer are styled:
	| type | style |
	| - | - |
	| `submit` | secondary (gray) |
	| `button` | secondary (gray) |
	| `reset` | tertiary (white) |

## UI

| S Form | C Form |
| - | - |
| <img width="992" alt="Screenshot 2023-09-13 at 2 31 17 PM" src="https://github.com/TACC/Core-Styles/assets/62723358/c8898f08-6fdd-406a-92fc-0a2112f12467"> | <img width="992" alt="Screenshot 2023-09-13 at 2 32 00 PM" src="https://github.com/TACC/Core-Styles/assets/62723358/c2cc8122-cf90-4f63-aaa1-34ae64e505bd"> |
